src/nvim/options.lua

```diff
@@ -2000,7 +2000,11 @@ return {
       vi_def=true,
       expand=true,
       varname='p_sh',
-      defaults={if_true={vi="sh"}}
+      defaults={
+        condition='WIN32',
+	if_true={vi="cmd.exe"},
+	if_false={vi="sh"}
```

Comment:
`condition` is aligned I think (two spaces) but the `if_` aren't (used a tab). Correct?